main.css

```diff
@@ -20,95 +40,68 @@ body{
   position:fixed;
   left: 700px;
   top: 250px;
-  z-index: -1;
-}
+  z-index: -1; */
+
 
 .container{
- 
-  
-  
-  
-  
-  background-color: #ffff;
-  border-radius: 10px 10px 10px 10px;
-} 
-
-.card{
-  display: 
-  background-color: #ffff;
-  flex-flow: column;
-  align-items: center;
-}
+  background-color:hsl(0, 0%, 100%);
+  text-align: center;
+  width: 350px;
+  border-radius: 15px;
+  overflow: hidden;
 
-.intro-one{
-  display:
-  flex-direction: row;
-  justify-content:flex-end;
-}
-.intro-two{
-  display: 
-  flex-direction: row;
-}
-.intro{
-  padding-top: 50px;
-  display: 
-  flex-direction: column;
-  align-items: center;
 }
 
-.age{
-  display: 
-  text-decoration: grey;
+.upper{
+  height: 150px;
+   width: 350px;
+   background: url(./doc_images/bg-pattern-card.svg);
+   background-size: cover;
+  
 }
 
-.info{
-  display: flex;
-  flex-direction: row;
-  justify-content: space-around;
-}
-.follow{
-  display: flex;
-  justify-items: baseline;
-  flex-direction: column;
-}
+#img{
```

Comment:
Can you please tell me why did you give the id to the image instead you may have used `img{ }` directly.

Reply:
This is because earlier you have added background images in html, so there is conflict between two img but after I added them through Css, so  I forgot to remove id_
but it is ok because any user will find easily through id

Reply:
if you want i will remove it

Reply:
Sounds good! Where did you get this code from?

Reply:
For html container format i take little reference from online websites in writen format 
and css is full my 
I tried every aspects to make page good

Reply:
can you send the images of before and after?

Reply:
you mean images of code

Reply:
yes

Reply:
ok